examples/fabric-admin/commands/interactive/InteractiveCommands.h

```diff
@@ -22,8 +22,6 @@
 #include "../common/CHIPCommand.h"
 #include "../common/Commands.h"
```

Comment:
Absolute paths for include are not great...

Reply:
I think you mean relative path, changed to #include <commands/common/Commands.h>